doc/docker/Dockerfile-solr

```diff
@@ -18,5 +20,11 @@ RUN mkdir -p /opt/solr/server/ez/template \
 # Set our core config as home
 ENV SOLR_HOME /opt/solr/server/ez
 
-# Make sure core is created on startup
-CMD ["solr-create", "-c", "collection1", "-d", "/opt/solr/server/ez/template"]
+# Make sure cores are created on startup
+RUN bin/solr -s ez -p $SOLR_PORT \
+ && bin/solr create_core -c collection1 -d server/ez/template -p $SOLR_PORT \
+ && bin/solr create_core -c econtent -d server/ez/template -p $SOLR_PORT \
+ && bin/solr create_core -c econtent_back -d server/ez/template -p $SOLR_PORT
```

Comment:
And you don't create them as ezcontent1 and ezcontent2 and renames them afterwards because that won't persist when solr is restarted, right?

If that is the case, then I think the idea of swapping the corenames in ezcommerce GUI is dangerous approach

Reply:
I do not create them as content1 and content2 in order to not complicate the docker setup. The target core names are econtent and econtent_back and according to one of our calls, creating them with these names directly does not matter for eZ Commerce - its mainly personal feeling what is confusing and what is not.

Reply:
@vidarl is this ok for now until commerce adjusts the approach?

Reply:
ping @vidarl

Reply:
I would prefer if the cores and directory naming is same inside docker container is the same as if using `install-solr.sh`.
But it is not a show shopper